deployment/ansible/roles/model-my-watershed.rwd/templates/upstart-mmw-rwd.conf.j2

```diff
@@ -0,0 +1,28 @@
+description "Rapid Watershed Delineation"
+
+start on (filesystem and started docker)
```

Comment:
I think that `start on` is going to have to be expanded to something like:

```bash
{% if ['development', 'test'] | some_are_in(group_names) -%}
start on (vagrant-mounted)
{% else %}
start on (filesystem and started docker)
{% endif %}
```

Because we don't want Docker to try and start before the Vagrant share is accessible.